app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(
+            self,
+            name: str
+    ) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError

```

Comment:
The exception `OutdatedVaccineError` is raised without a descriptive message. According to the checklist, you should provide a descriptive error message when raising exceptions. Consider adding a message like `raise OutdatedVaccineError('Visitor's vaccine is expired')`.